### PR TITLE
Add more improvised tool options

### PR DESCRIPTION
**Pull request by Rail-Runner**
#### Summary
```SUMMARY: Content "Add more improvised tool options"```

#### Purpose of change

Allows player to use many items as improvised, mostly low-quality, tools, including, but not limited to, using shovels and sheet metal for cooking meat on them, bricks, crowbars and other heavy objects for hammering, and axes for butchering. I'm not entirely sure on some of them, so explainations on why some of them wouldn't work as a tool are welcome.

Also, sealed tin cans and aluminum cans no longer have qualities of unsealed ones; they have to be unsealed first. Doesn't entirely solve the issue of being able to use non-empty containers as tools just as if they were empty, but probably still better than nothing.

#### Describe the solution
Simply adds tool qualities to many items, so that they can be used as tools.

### CODE REVIEW

File: data/json/items/armor.json [hunk 1]
```diff
@@ -5207,6 +5207,7 @@
     "warmth": 5,
     "material_thickness": 2,
     "container_data": { "contains": 40, "seals": true, "watertight": true },
+    "qualities": [ [ "BOIL", 1 ] ],
```
Comment (sfsworms) [inline data/json/items/armor.json]: Boil stuff in a plastic jerrycan?

Comment (Rail-Runner) [reply data/json/items/armor.json]: Oh, thought it was made from the metal one. Will remove.

**Comment by Maddremor** [comment]
I'm a bit sceptical about boiling things on a flat dinnerplate and butchering with a turned off chainsaw. Are you rubbing the chains against the carcass?

**Comment by Rail-Runner** [comment]
Just checked the chainsaws, only turned on ones can be used for butchering. As for dinner plates, I thought of them as being large enough to boil a small amount of liquid in.

**Pull request merged**